pkgs/development/python-modules/face-recognition/default.nix

```diff
@@ -36,6 +38,9 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  # Disables tests when running with cuda due to https://github.com/NixOS/nixpkgs/issues/225912
+  doCheck = !config.cudaSupport;
```

Comment:
Note for future: I think it's more flexible to expose a passthru in `dlib` and test that intead of the global `config`, because one might override `dlib = prev.dlib.override { cudaSupport = true; }` specifically. But this is good enough as it is IMO